app/assets/stylesheets/application.scss

```diff
@@ -30,6 +30,27 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+
+ .green-completion{
+  color: white;
+  width: 85px;
+  background-color: green;
+  margin: 5px 5px 5px 5px;
+  border-width: 1px;
+  border-style: solid;
+  border-color: rgba(0,0,0,.2);
+ }
+
+.red-completion {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background-color, border-color, border-style, border-width, color, margin, width